kustomize/v1alpha7/flatcar/patch-flatcar.yaml

```diff
@@ -17,9 +17,16 @@ spec:
     joinConfiguration:
       nodeRegistration:
         name: $${COREOS_OPENSTACK_HOSTNAME}
+        kubeletExtraArgs:
+          provider-id: null
     initConfiguration:
       nodeRegistration:
         name: $${COREOS_OPENSTACK_HOSTNAME}
+        kubeletExtraArgs:
+          # Fixme(lentzi90): This is here just to override the value set in the default
+          # kustomization. It will be replaced with a value that works for flatcar in
+          # https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1564
```

Comment:
You added this comment in both places other than the one where I asked for it 🤣 I'll try not to take it personally.

It's not worth updating imho. The important thing is that that somebody looking will see the comment, and they'll see one of these.

Reply:
OMG sorry! I'm not sure how I managed to miss the first occurance :facepalm: :joy: